core/tests/webdriverio/libraryFlow.js

```diff
@@ -72,7 +72,8 @@ describe('Library pages tour', function() {
   it('should visit the search page', async function() {
     await libraryPage.get();
     await libraryPage.findExploration(SEARCH_TERM);
-    expect(await browser.getCurrentUrl()).toContain('search/find?q=python');
+    await waitFor.urlToBe(
+      'http://localhost:9001/search/find?q=python&language_code=(%22en%22)');
```

Comment:
`getCurretnUrl` is not working because it doesn't wait till the url contains the given string so I need to explicitlty wait till the url contains the given string.